build.gradle

```diff
@@ -19,6 +20,11 @@ tasks.withType(JavaCompile).configureEach {
 	it.options.release = 17
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+	kotlinOptions {
+		jvmTarget = "16" // Kotlin doesnt support 17!?
```

Comment:
1.6.0+ (most recent version is `1.6.10`) is required for java 17

Reply:
Ah, ok. We are tied to the version of kotlin gradle uses here sadly. The latest nightly uses 1.6.10 but not 7.3 this seems to work but does output a warning.